webpack/assets/javascripts/react_app/components/Pagination/PaginationRow.js

```diff
@@ -0,0 +1,77 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import Pagination from './Pagination';
+import { Paginator } from 'patternfly-react';
+
+const PaginationRow = (props) => {
+  const {
+    onPageSet,
+    onPerPageSelect,
+    onChange,
+    pagination,
+    dropdownButtonId,
+    itemCount,
+    ...otherProps,
+  } = props;
+
+  const onPageSetUpdate = (page) => {
+    update({ page });
+    onPageSet(page);
+  };
+
+  const onPerPageSelectUpdate = (perPage) => {
+    update({ perPage, page: 1 });
+    onPerPageSelect(perPage);
+  };
+
+  const update = (changes) => {
+    const newPagination = { ...props.pagination, ...changes };
+
+    onChange({
+      page: newPagination.page,
+      perPage: newPagination.perPage,
+    });
+  };
+
+  const initPagination = (paginationFromProps) => {
+    const defaultPagination = {
+      page: 1,
+      perPage: 20,
+      perPageOptions: [5, 10, 15, 20, 25, 50],
+    };
+    return { ...defaultPagination, ...(paginationFromProps || {}) };
+  };
+
+  return (
+    <Pagination
+      data={{}}
```

Comment:
Why is data empty? how is `viewType, itemCount` being passed to `Pagination`? looks like its getting passed by `...otherProps` which is not ideal, you should just create a data object in your wrapper for more detailed propTypes 

```suggestion
      data={{ itemCount }}
```